tests/migration_test.go

```diff
@@ -309,26 +309,27 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		By("Waiting until the Migration Completes")
 
 		uid := ""
-		Eventually(func() bool {
+		Eventually(func() v1.VirtualMachineInstanceMigrationPhase {
 			migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(migration.Name, &metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			Expect(migration.Status.Phase).NotTo(Equal(v1.MigrationSucceeded))
+			phase := migration.Status.Phase
+			Expect(phase).NotTo(Equal(v1.MigrationSucceeded))
 
 			uid = string(migration.UID)
-			if migration.Status.Phase == v1.MigrationFailed {
-				return true
-			}
-			return false
+			return phase
 
-		}, timeout, 1*time.Second).Should(BeTrue())
+		}, timeout, 1*time.Second).Should(Equal(v1.MigrationFailed))
 		return uid
 	}
 
 	runStressTest := func(expecter expect.Expecter) {
 		By("Run a stress test to dirty some pages and slow down the migration")
 		_, err = expecter.ExpectBatch([]expect.Batcher{
-			&expect.BSnd{S: "stress --vm 1 --vm-bytes 800M --vm-keep --timeout 1600s&\n"},
+			&expect.BSnd{S: "\n"},
+			&expect.BExp{R: "\\#"},
+			&expect.BSnd{S: "stress --vm 1 --vm-bytes 800M --vm-keep --timeout 1600s&\n\n"},
```

Comment:
why double \n ?

Reply:
Because it is running in the background, try it locally and see one is not enough.

Reply:
I can't confirm that this is needed. This may depend on the shell in use. I am not sure I saw the test ever failing there, but worst case it makes the test more robust.